test/fact_importer_test_helper.rb

```diff
@@ -7,7 +7,7 @@ def allow_transactions_for(importer)
   end
 
   def allow_transactions_for_any_importer
-    FactImporter.importers.values.each do |importer|
+    Foreman::Plugin.fact_registry.importers.values.each do |importer|
```

Comment:
@ShimShtein should I do something like `Foreman::Plugin.fact_registry.importers.values.map(&:constantize).each do |importer|` ?

Reply:
Yes, makes sense.